dev-db/sqlmap/sqlmap-1.6.2.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{9..10} )
+PYTHON_REQ_USE="sqlite"
+
+inherit bash-completion-r1 python-single-r1 wrapper
+
+DESCRIPTION="An automatic SQL injection and database takeover tool"
+HOMEPAGE="https://sqlmap.org/"
+
+if [[ ${PV} == "9999" ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/sqlmapproject/sqlmap"
+else
+	SRC_URI="https://github.com/sqlmapproject/sqlmap/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="GPL-2"
```

Comment:
sqlmap itself is `GPL-2+`, and there are also vendored libraries in the `thirdparty` directory with their own licenses.

```suggestion
# sqlmap (GPL-2+)
# ansitrm (BSD)
# beautifulsoup (BSD)
# bottle (MIT)
# charder (LGPL-2.1+)
# clientform (BSD)
# colorama (BSD)
# fcrypt (BSD-2)
# identitywaf (MIT)
# keepalive (LGPL-2.1+)
# magic (MIT)
# multipartpost (LGPL-2.1+)
# ordereddict (MIT)
# prettyprint (BSD-2)
# pydes (public-domain)
# six (MIT)
# socks (BSD)
# termcolor (BSD)
# wininetpton (public-domain)
LICENSE="BSD BSD-2 GPL-2+ LGPL-2.1+ MIT public-domain"
```